lib/ui/src/chat/CodeBlocks.tsx

```diff
@@ -8,13 +8,19 @@ import { CopyButtonProps } from '../Chat'
 
 import styles from './CodeBlocks.module.css'
 
+export type CodyRecipesWidgetWrapper = React.ComponentType<{
+    targetRef: RefObject<HTMLElement>
+    children: React.ReactNode
+}> | null
```

Comment:
Nit: If the prop is optional, what if we simplify this type to always be non-null:

```suggestion
export type CodyRecipesWidgetWrapper = React.ComponentType<{
    targetRef: RefObject<HTMLElement>
    children: React.ReactNode
}>
```

And then just assign a default value of `null` in the component prop destructuring?